python/ray/serve/tests/test_model_wrappers.py

```diff
@@ -208,6 +208,36 @@ def test_model_wrappers_in_pipeline(serve_instance):
     return resp.json() == {"value": [42], "batch_size": 1}
 
 
+def test_model_wrapper_reconfigure(serve_instance):
+    path = tempfile.mkdtemp()
+    uri = f"file://{path}/test_uri"
+    Checkpoint.from_dict({"increment": 2}).to_uri(uri)
+
+    predictor_cls = "ray.serve.tests.test_model_wrappers.AdderPredictor"
+
+    with InputNode() as dag_input:
+        m1 = ModelWrapperDeployment.bind(
+            predictor_cls=predictor_cls,
+            checkpoint=uri,
+        )
+        new_predictor_cls = m1.reconfigure(uri, predictor_cls)
```

Comment:
maybe create another checkpoint here to make sure we are actually using a different increment?